host/xtest/xml/include/xml_crypto_api.h

```diff
@@ -374,7 +355,11 @@ static const uint8_t NONCE1_VALUE_AES_CCM[] = {
 
 /*ALL_CRYPTO_DATA_VALUE*/
 static const uint8_t DATA_FOR_CRYPTO1[] = {
-	0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0A, 0x0B,
+	/*
+	 * do not start we a '0' as this is the representation of a number, so
+	 * the most significant '0' are removed after encrypt / decrypt
+	 */
+	0x01, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0A, 0x0B,
```

Comment:
I don't think it's OK. We should be able to encrypt any data buffer! At least I see no restriction in the spec for `TEE_AsymmetricEncrypt()`. I think this needs further investigation.